cadasta/gui/tools/wizard/step_project_creation01.py

```diff
@@ -50,7 +53,23 @@ def __init__(self, parent=None):
 
     def set_widgets(self):
         """Set all widgets on the tab."""
-        return
+        contacts = Contact.get_rows()
+
+        for contact in contacts:
+            contact_string = contact.name + ((
```

Comment:
i think we could bring out if else?
contact_email = (contact.email) if contact.email else ''
contact_phone = (contact.phone) if contact.phone else ''